rootfs/etc/nginx/lua/util.lua

```diff
@@ -53,6 +53,16 @@ function _M.get_first_value(var)
   return t[1]
 end
 
+-- splits strings into host and port
+function _M.parse_addr(addr)
```

Comment:
I'd rather have `encode_endpoint(endpoint)` and `decode_endpoint(endpoint)` functions here - we already have a structure to hold endpoint object, we don't need to introduce a new one